testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -1019,3 +1019,13 @@ def test_missing_icodes_range_VE(self, u):
         u = make_Universe(('resids',))
         with pytest.raises(ValueError):
             u.select_atoms('resid 10A-12')
+
+class TestArbitraryAtomGroupSelection:
```

Comment:
classes should inherit from `object` to avoid different behaviour in 2/3, it wouldn't really make a difference here but it's a good habit

Reply:
Okay. I was wondering why the previous tests had old-style classes. Thank you for clarifying.